rust/operator-binary/src/controller.rs

```diff
@@ -62,26 +73,17 @@ use stackable_operator::{
         statefulset::StatefulSetConditionBuilder,
     },
     time::Duration,
+    utils::COMMON_BASH_TRAP_FUNCTIONS,
+};
+use std::{
+    borrow::Cow,
+    collections::{BTreeMap, HashMap},
+    ops::Deref,
+    sync::Arc,
```

Comment:
Those imports should be at the top of the file.

Reply:
https://github.com/stackabletech/nifi-operator/pull/528/commits/8ac905c656eec36cf0b343d19ddc0a6a2a84a8e5